test/jest/secureRoute.test.tsx

```diff
@@ -305,7 +305,8 @@ describe('<SecureRoute />', () => {
         </MemoryRouter>
       );
       const secureRoute = wrapper.find(SecureRoute);
-      expect(secureRoute.find(Route).props().exact).toBe(true);
+      const props: RouteProps = secureRoute.find(Route).props();
+      expect(props.exact).toBe(true);
```

Comment:
To prevent this:
```
ts-jest[ts-compiler] (WARN) test/jest/secureRoute.test.tsx:308:46 - error TS2339: Property 'exact' does not exist on type 'Readonly<{}> & Readonly<{ children?: ReactNode; }>'.
308       expect(secureRoute.find(Route).props().exact).toBe(true);
                                                 ~~~~~
test/jest/secureRoute.test.tsx:325:46 - error TS2339: Property 'strict' does not exist on type 'Readonly<{}> & Readonly<{ children?: ReactNode; }>'.
325       expect(secureRoute.find(Route).props().strict).toBe(true);
                                                 ~~~~~~
test/jest/secureRoute.test.tsx:342:46 - error TS2339: Property 'sensitive' does not exist on type 'Readonly<{}> & Readonly<{ children?: ReactNode; }>'.
342       expect(secureRoute.find(Route).props().sensitive).toBe(true);
```